server/config/middlewares.js

```diff
@@ -21,15 +22,16 @@ function getKeyRing(){
           passphrase: passphrase
         }, function(err) {
           if (!err) {
-            console.log('Loaded private key with passphrase');
+            logger.prod('Loaded private key with passphrase');
           } else {
-            console.log('Err:', err);
+            logger.error('Error unlocking pgp');
+            logger.warn('Error unlocking pgp:', err);
             process.exit();
           }
 
         });
       } else {
-        console.log('Loaded private key w/o passphrase');
+        logger.prod('Loaded private key w/o passphrase');
```

Comment:
I think when in production we should default to showing security related config information on startup to help avoid configuration mistakes.

Things like, are we routing via Mailgun (less privacy), and are we loading a key that doesn't have a passphrase (probably a bad idea)